### PR TITLE
Fix duplicate `CLEARML__secure__auth__token_secret` in `apiserver-asyncdelete-deployment.yaml`

### PR DESCRIPTION
**Pull request by benglewis**
**What this PR does / why we need it**:
Fixes duplicate environment variable

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/allegroai/clearml-helm-charts/blob/main/CONTRIBUTING.md#pull-requests) guide (**required**)
- [x] Verify the work you plan to merge addresses an existing [issue](https://github.com/allegroai/clearml-helm-charts/issues) (If not, open a new one) (**required**)
- [x] Check your branch with `helm lint` (**required**)
- [x] Update `version` in `Chart.yaml` according [semver](https://semver.org/) rules (**required**)
- [x] Substitute `annotations` section in `Chart.yaml` annotating implementations (useful for Artifecthub changelog) (**required**)
- [x] Update chart README using [helm-docs](https://github.com/norwoodj/helm-docs) (**required**)

**Which issue(s) this PR fixes**:

Fixes #<issue number>

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by benglewis** [comment]
@filippo-clearml Is there any chance that this will be merged? It is blocking me upgrading to the latest Helm chart with the latest ClearML :/

**Comment by benglewis** [comment]
@filippo-clearml @valeriano-manassero I don't mean to be a bugbear, but we are stuck unable to upgrade our ClearML version because of this bug and so I would really like to see this PR merged 🙏

**Comment by filippo-clearml** [comment]
Hey @benglewis sorry for the delay.
Of course, we're going to merge this soon! Thanks for the contribution :)

File: charts/clearml/Chart.yaml
Comment (filippo-clearml) [inline charts/clearml/Chart.yaml:1]: @benglewis - Could you please edit the lines 35:36 to something meaningful like
```
kind: fixed
description: Removed duplicated env 
```

**Comment by benglewis** [comment]
@filippo-clearml I applied the change to `Chart.yaml`. Hopefully you can merge this :)

**Pull request merged**